config/routes.rb

```diff
@@ -16,6 +16,8 @@
   
   resources :media,
     only: [:show]
-
+  
+  resources :warm_cache,
+    only: [:index]
   
```

Comment:
@mccalluc feel free to use more semantic routes and controller method names when the default resourceful route names aren't as applicable.

For instance, maybe something like...
```ruby
get 'warm_cache', to: 'cache#warm'
```

Just a suggestion anyway. I think the main benefits to resourceful routing comes if you have CRUD needs, if you have nested routes, or if you want all the automagically generated url helpers.